.chain-interactions/send-transactions/interoperability/transfer-assets-into-polkadot.md

```diff
@@ -0,0 +1,221 @@
+
+This guide walks you through transferring tokens from Ethereum to Polkadot using the [ParaSpell XCM SDK](/reference/tools/paraspell/){target=\_blank} and [Snowbridge](https://docs.snowbridge.network/){target=\_blank}. Snowbridge is a trustless, decentralized bridge integrated into the Polkadot protocol that enables secure asset transfers between Ethereum and Polkadot ecosystems.
+
+### How the Bridge Works
+
+The following diagram shows the complete flow when bridging WETH from Ethereum to Polkadot:
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant WETH as WETH Contract
+    participant Gateway as Snowbridge Gateway
+    participant Relayers
+    participant BridgeHub as Bridge Hub
+    participant PolkadotHub as Polkadot Hub
+
+    User->>WETH: 1. Approve Gateway to spend WETH
+    WETH-->>User: Approval confirmed
+    User->>Gateway: 2. Call sendToken(WETH, amount, destination)
+    Gateway->>WETH: Transfer WETH from user
+    Gateway->>Gateway: Lock tokens & emit OutboundMessageAccepted
+    Gateway-->>User: Ethereum tx confirmed
+    Note over Relayers: ~30 minutes
+    Relayers->>BridgeHub: 3. Relay message to Polkadot
+    BridgeHub->>BridgeHub: Verify Ethereum consensus proof
+    BridgeHub->>PolkadotHub: 4. Forward via XCM
+    PolkadotHub->>PolkadotHub: Mint wrapped WETH as foreign asset
+    PolkadotHub-->>User: 5. Tokens available in account
+```
+
+In this guide, you will:
+
+- Set up an Ethereum development environment with the ParaSpell SDK
+- Approve tokens for the Snowbridge Gateway contract
+- Build and execute a bridge transfer from Ethereum to Polkadot Hub
+- Monitor the transfer status
+
+### Prerequisites
+
+Before you begin, ensure you have the following:
+
+- A basic understanding of [XCM](/parachains/interoperability/get-started/){target=\_blank}
+- Familiarity with JavaScript/TypeScript and Ethereum development
+- [Node.js](https://nodejs.org/){target=\_blank} v18 or higher and npm installed
+- An Ethereum wallet with WETH tokens (see next section to wrap ETH)
+- A Polkadot account to receive the bridged assets
+
+## Prepare Tokens for Bridging
+
+To bridge ETH, you need to wrap it into WETH first. The WETH contract on Ethereum mainnet is:
+
+```
+0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
+```
+
+!!!note "Why WETH instead of ETH?"
+    Snowbridge only supports ERC-20 tokens—it cannot bridge native ETH directly. The bridge uses a standardized token interface to lock assets on Ethereum and mint corresponding representations on Polkadot. Since native ETH doesn't conform to the ERC-20 standard, you must first wrap it into WETH (Wrapped ETH), which is an ERC-20-compliant token pegged 1:1 with ETH.
+
+You can wrap ETH by:
+
+1. Visiting the [WETH contract on Etherscan](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2#writeContract){target=\_blank}
+2. Connecting your wallet
+3. Calling the `deposit` function with the amount of ETH you want to wrap
+
+## Initialize Your Project
+
```

Comment:
```suggestion

Follow these steps to setup and initialize your project:
```

Unless we've moved away from it (someone update me if that's the case please), we should be using numbered steps when we are giving the user things they have to do in a specific order to be successful.